app/MMB/src/scenes/tripServices/insurance/InsuranceTermsPdfScene.js

```diff
@@ -0,0 +1,45 @@
+// @flow strict
+
+import * as React from 'react';
+import idx from 'idx';
+import { PrivateApiRenderer, graphql } from '@kiwicom/mobile-relay';
+import PdfViewer from '@kiwicom/mobile-pdf';
+
+import type { InsuranceTermsPdfSceneQueryResponse as Props } from './__generated__/InsuranceTermsPdfSceneQuery.graphql';
+
+const renderPdfViewer = (props: Props) => {
+  const documents = idx(props, _ => _.allDocuments.edges) || [];
+  let document;
+  let uri = '';
+  if (documents.length >= 1) {
+    document = idx(documents[0], _ => _.node);
+  }
```

Comment:
this if is not needed, idx can handle this for you
`const document = idx(documents, _ => _[0].node)`

Reply:
Ah, cool!